android/app/src/main/java/org/electroncash/electroncash3/ColdLoad.kt

```diff
@@ -69,4 +134,72 @@ class ColdLoadDialog : AlertDialogFragment() {
             dismiss()
         } catch (e: ToastException) { e.show() }
     }
+
+
+    /**
+     * Check if a loaded transaction is signed.
+     * Displays the signing status below the raw TX field.
+     * (signed, partially signed, empty or invalid)
+     */
+    private fun updateStatusText(tx: PyObject) {
+        try {
+            val txInfo = daemonModel.wallet!!.callAttr("get_tx_info", tx)
+
+            // Check if the transaction can be processed by this wallet or not
+            if (txInfo["amount"] == null) {
+                idTxStatus.setText(R.string.transaction_unrelated)
```

Comment:
Can you tell me how you managed to get "unsigned" and "partially signed" status texts in unrelated wallets? I've tried with multiple unsigned and partially signed transactions, and used unrelated standard and multisig wallets; I get the "unrelated transaction" status every time.

I agree with the second point, and made it possible to broadcast a fully signed transaction from any wallet.

Reply:
Ah, it looks like this actually happens when the *receiving* address of the transaction belongs to the current wallet, but the sending address(es) don't. In that case, the transaction is "related to" the wallet, but the user still can't sign it. So this probably had nothing to do with multisig versus standard at all, it just looked that way because of how I was testing it.

This isn't very likely to happen in the real world, and the desktop app behaves the same way, so let's just leave it.